### DISCUSSION TITLE
How do you understand the ability to 'analyze', and where is 'analysis' reflected in the G6?

### DISCUSSION THREAD

**Discussion by liaodalin19903** [antvis/G6#6381] [Q&A] [answered]
> G6 是一个图可视化引擎。它提供了图的绘制、布局、分析、交互、动画等基础的图可视化能力。旨在让关系变得透明，简单。让用户获得关系数据的 Insight。

请问如何理解`分析`能力，`分析`在G6中哪些地方有得到体现呢？

**Comment by liaodalin19903** [comment]
图分析能力：是指能够对图中的数据（包括节点、边以及它们之间的关系）进行深入研究和理解的功能。
它可以帮助用户发现隐藏在复杂关系网络中的模式、规律、异常等信息。

例如，通过分析可以找出图中节点的重要性排序，或者发现某些节点之间频繁出现的连接模式，这类似于在社交网络分析中找出关键意见领袖或者紧密的社交圈子。

**Reply by hustcc** [reply, marked as answer]
可以理解基于 G6 可以做图分析需要的技术能力，比如节点边演示，布局，交互等。在 GraphInsight 中以工具的形式透出。
